src/librawspeed/decoders/Cr2Decoder.cpp

```diff
@@ -273,11 +273,10 @@ RawImage Cr2Decoder::decodeNewFormat() {
 
 RawImage Cr2Decoder::decodeRawInternal() {
   try {
-    if (hints.find("old_format") != hints.end()) {
+    if (hints.find("old_format") != hints.end())
       return decodeOldFormat();
-    } else {
-      return decodeNewFormat();
-    }
+
+    return decodeNewFormat();
```

Comment:
> Hmm, but it helps to write spaghetti-like code, code with unneeded {}

If you are calling that code of mine "spaghetti-like" because there are more `{}` than necessary: please note that I merely used them here because is seemed to be the preferred style in the rest of the code base. I personally prefer to have them removed (and since I noticed, you are also not strictly following the existing example, I started to move back to my own preference ;))

> In this Cr2Decoder::decodeRawInternal() case i agree,

Nice. Even though: please don't bother changing that line here. My next pr will simply remove that whole block of code anyway :).

> but in general, i'm not so sure it wouldn't be be a positive to discourage it...

In general, I also think it is better to discourage it. In the Huffman code below I thought it would be more understandable the way I wrote it but is admittedly a border line situation.

=> from my side: please merge.